templates/common/alibabacloud/files/usr-local-bin-alibaba-kubelet-nodename.yaml

```diff
@@ -5,10 +5,10 @@ contents:
     #!/bin/bash
     set -e -o pipefail
```

Comment:
I'd use full [bash strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) as a general rule, specifically we want `-u` here.